docker-compose.yml

```diff
@@ -445,6 +445,7 @@ services:
       - COUCHBASE_HOST=couchbase
       - COUCHBASE_PORT=8091
       - ASB_CONNECTION_STRING=Endpoint=sb://azureservicebus-emulator:5672;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=SAS_KEY_VALUE;UseDevelopmentEmulator=true;
+      - TEST_AGENT_HOST=test-agent
```

Comment:
Is this what caused the snapshots to change?

Reply:
Yesss instead of getting them from the Mock Agent, the test Agent returns JSON instead so they look different but same amount of metrics with same data within, we used the changes from the version below:

https://github.com/DataDog/dd-apm-test-agent/releases/tag/v1.32.0